src/components/pages/profile/Profile.tsx

```diff
@@ -15,15 +15,19 @@ const SettingsPage = () => {
   const { addToast } = useToast();
 
   const getUserInfo = async () => {
-    const response = await getMemberAssociatedWithToken();
-    const initalValues = {
-      name: response.realName ?? '',
-      email: response.email ?? '',
-      classof: response.classof ?? '',
-      phone: response.phone ?? '',
-    };
-    setInit(initalValues);
-    setActive(response.status);
+    try {
+      const response = await getMemberAssociatedWithToken();
+      const initalValues = {
+        name: response.realName ?? '',
+        email: response.email ?? '',
+        classof: response.classof ?? '',
+        phone: response.phone ?? '',
+      };
+      setInit(initalValues);
+      setActive(response.status);
+    } catch (error) {
+      console.log(error);
```

Comment:
Gracefully ignore error? :angel:

Reply:
yes, can be fixed if it turns out someone got to a protected page without being logged in